dist/js/util.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
Does this "exports" property need to be added on every JS file consuming it ? 
If we are going to create a single JS file to be included in the html page then we can get rid of the multiple declaration of this property.